apm-sniffer/apm-agent-core/src/main/java/org/apache/skywalking/apm/agent/core/util/MethodUtil.java

```diff
@@ -42,4 +43,21 @@ public static String generateOperationName(Method method) {
         operationName.append(")");
         return operationName.toString();
     }
+
+
+    public static boolean isMethodExist(ClassLoader classLoader, String className, String methodName, Class<?>... parameterTypes) {
+        try {
+            Class<?> httpServletResponse = Class.forName(className, true, classLoader);
```

Comment:
Wrong variable name

Reply:
Parameter type should be String if class name is string.

Reply:
Yes,I forgot update